src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1765,14 +1765,21 @@ void window_guest_rides_paint(rct_window* w, rct_drawpixelinfo* dpi)
     screenCoords.y = w->windowPos.y + window_guest_rides_widgets[WIDX_PAGE_BACKGROUND].bottom - 12;
 
     auto ft = Formatter::Common();
-    ft.Add<rct_string_id>(STR_PEEP_FAVOURITE_RIDE_NOT_AVAILABLE);
     if (peep->FavouriteRide != RIDE_ID_NULL)
```

Comment:
I wonder if we even need this check. Could simplify the if else if get_ride returns nullptr on ride id null.

Reply:
Sounds like a good idea, then the two identical else blocks can be merged too.

Reply:
Just checked and it should work fine to make that change.